### PR TITLE
Replace deprecated CVE_CHECK_IGNORE with CVE_STATUS

### PR DESCRIPTION
**Pull request by yumiweidemao**
Per [Yocto manual](https://docs.yoctoproject.org/ref-manual/variables.html#term-CVE_CHECK_IGNORE), CVE_CHECK_IGNORE has been deprecated and should be replaced by CVE_STATUS.

When building with openembedded-core, there is a fallback mechanism in [cve-check.bbclass](https://github.com/openembedded/openembedded-core/blob/master/meta/classes/cve-check.bbclass#L102) but it will print some noisy warning messages.

This pull request is to replace the deprecated CVE_CHECK_IGNORE with CVE_STATUS.

### CODE REVIEW

**Review by ichergui — APPROVED** [review]
LGTM. Thanks @yumiweidemao

**Review by ichergui — CHANGES_REQUESTED** [review]
@yumiweidemao  Could you please add `Signed-off-by` ? So, we can merge the PR 
Thanks

**Comment by yumiweidemao** [comment]
@ichergui Signoff added. Thanks!

File: recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc [hunk 1]
```diff
@@ -126,4 +126,4 @@ EXTRA_OECONF_PATHS = "\
 "
 
 # Is a binutils 2.26 issue, not gcc
-CVE_CHECK_IGNORE += "CVE-2021-37322"
+CVE_STATUS[CVE-2021-37322] = "ignored"
```
Comment (madisongh) [inline recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc:129]: This should specify the actual reason. Looking at the setting in OE-Core's gcc recipe, it should be `"cpe-incorrect: Is a binutils 2.26 issue, not gcc"`.

**Review by madisongh — CHANGES_REQUESTED** [review]

File: recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc (hunk 1 above)
Comment (yumiweidemao) [reply recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc:129]: Modified the comment

Comment (madisongh) [reply recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc:129]: It's the setting that needs changing, and the comment should be removed.  See corresponding line in the [current gcc recipe](https://git.yoctoproject.org/poky/tree/meta/recipes-devtools/gcc/gcc-14.2.inc#n111).

Comment (yumiweidemao) [reply recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc]: Modified the line. Thanks for the clarification!

**Pull request merged**